internal/service/timestreaminfluxdb/db_instance.go

```diff
@@ -494,6 +519,10 @@ func (r *resourceDBInstance) Update(ctx context.Context, req resource.UpdateRequ
 		plan.SecondaryAvailabilityZone = flex.StringToFrameworkLegacy(ctx, output.SecondaryAvailabilityZone)
 	}
 
+	if plan.SecondaryAvailabilityZone.IsUnknown() {
+		plan.SecondaryAvailabilityZone = state.SecondaryAvailabilityZone
```

Comment:
This is the fix for `secondary_availability_zone` being left as unknown when tags are updated.

From what I can determine, the problem is that `Update` is called when tags are updated (for some reason) but `UpdateDbInstance` isn't actually called, as `UpdateDbInstance` doesn't allow tags to be updated, therefore `secondary_availability_zone` is left as unknown. I would like a better fix that scaled better for possible future computed values, but at this point I'm not sure what that better fix would be.

Reply:
Added.